.env.build

```diff
@@ -0,0 +1 @@
+SERVER_URL="https://c0d3.com"
```

Comment:
This should be removed from the repo since it's already been gitignored. 

Try running `git rm .env.build --cached`. That should remove it from the git repository and keep it on your computer.